src/Common/Core/Impl/Extensions/IOExtensions.cs

```diff
@@ -84,5 +84,16 @@ public static class IOExtensions {
             }
             return path.Length;
         }
+
+        public static string EnsureTrailingSlash(this string path) {
+            if (!string.IsNullOrEmpty(path)) {
+                char slash = path.IndexOf(Path.AltDirectorySeparatorChar) >= 0 ? Path.AltDirectorySeparatorChar : Path.DirectorySeparatorChar;
+                if (path[path.Length - 1] != slash) {
```

Comment:
Yes.

---
In reply to: [80995728](https://github.com/Microsoft/RTVS/pull/2408#discussion_r80995728) [](ancestors = 80995728)